modules/nw-externalip-about.adoc

```diff
@@ -51,7 +51,7 @@ If routed correctly, external traffic from the configured external IP address bl
 
 [IMPORTANT]
 ====
-You must ensure that the IP address block you assign terminates at one or more nodes in your cluster.
+As a cluster Administrator, it is your responsibility to configure routing to externalIPs on both SDN and OVN. You must also ensure that the IP address block you assign terminates at one or more nodes in your cluster.
```

Comment:
A slight suggestion; I don't know if you provide links to other repos, but you could add https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/external-ip-and-loadbalancer-ingress.md#guidance-for-administrators OR https://kubernetes.io/docs/concepts/services-networking/service/#external-ips as external link where users can read more.